templates/partials/footer.hbs

```diff
@@ -1,31 +1,72 @@
   </article>
 
   <footer>
-    <a href="/">Add test</a> &middot;
-
-    {{!-- TODO: this is never added to the context, but set on session after test added
-    https://github.com/mtharrison/hapi-context-credentials/blob/master/index.js
-    --}}
-    {{#if authorSlug}}
-      <a href="/browse/{{authorSlug}}">My tests</a> &middot;
-    {{/if}}
-
-    <a href="/browse">Latest</a> &middot;
-
-    <a href="/popular">Popular</a> &middot;
-
-    <a href="/faq">FAQ</a> &middot;
-
-    <a href="/faq#donate">Donate</a> &middot;
-
-    <a href="https://twitter.com/jsperf" rel="nofollow">twitter: @jsperf</a> &middot;
-
-    <a href="https://github.com/jsperf/jsperf.com">source on GitHub</a> &middot;
-
-    <a href="https://benchmarkjs.com/">Benchmark.js</a> &middot;
-
-    by <a href="https://mathiasbynens.be/">@mathias</a> and <a href="/contributors">contributors</a>
-
+    <ul>
+      <li>
+        <h4>Sponsors</h4>
+        <p>Hosting generously provided by <a href="https://m.do.co/c/b426e15331e8" target="_blank">DigitalOcean</a> and <a href="https://zeit.co/" target="_blank">Zeit</a></p>
```

Comment:
Where did this link come from?

FTR: DigitalOcean gave us some much-appreciated credits, but they don’t provide us with unlimited hosting forever. That’s why there is still a necessity for a “hosting fund”. We should probably figure out how we’ll make this work. Where (i.e. which donations) do the hosting credits come from?